.gitignore

```diff
@@ -44,6 +44,7 @@ nosetests.xml
 coverage.xml
 *,cover
 .hypothesis/
+test.conf
```

Comment:
What makes test.conf? We should fix what is not cleaning it up ... can be in a follow up diff.

Reply:
it's a product of `tox`